src/main/java/com/easypost/model/CreditCard.java

```diff
@@ -0,0 +1,11 @@
+package com.easypost.model;
+
+/**
+ * CreditCard is a model class that represents a credit card.
+ *
+ * @deprecated Use {@link com.easypost.model.PaymentMethodObject} instead.
+ * Deprecated: v5.5.0 - v7.0.0
+ */
+@Deprecated
+public class CreditCard extends BaseCreditCard {
```

Comment:
I thought we were going to remove the CreditCard class?

Reply:
Awww, rebase gone wrong (had 100+ merge conflicts)